src/main/java/com/google/sps/models/RecommendedCategories.java

```diff
@@ -0,0 +1,61 @@
+package com.google.sps.servlets;
+import java.util.ArrayList;
+import java.util.List;
+
+public class RecommendedCategories extends Cluster {
+    private String title = "Browse recommended categories";
+    private String subtitle = "Didn't find what you're looking for?";
+    private List<Container> container = new ArrayList<Container>();
+    private String navigation = "See all categories";
+
+    RecommendedCategories() {
+        makeCluster();
+        this.type = getType();
+        this.id = getID();
+    }
+
+    public String getTitle() {
+        return this.title;
+    }
+
+    public String getSubtitle() {
+        return this.subtitle;
+    }
+
+    public List<Container> getContainer() {
+        return this.container;
+    }
+
+    public String getNavigation() {
+        return this.navigation;
+    }
+
+    public String getType() {
+        return "RecommendedCategories";
+    }
+
+    public String getID() {
+        return "recommendedCategoriesID";
+    }
+
+    public void makeCluster() {
+        Container mockContainer1 = new Container("mockContainer1", "Health & fitness", "mockIcon1.png");
```

Comment:
I think we should rename this to something more specific. Container is a bit too generic. How about RecommendedCategoryLink or RecommendedCategoryCard